src/Illuminate/Mail/Mailable.php

```diff
@@ -759,6 +759,25 @@ public function with($key, $value = null)
         return $this;
     }
 
+    /**
+     * Apply the callback's message changes if the given "value" is true.
+     *
+     * @param  mixed  $value
+     * @param  callable  $callback
+     * @param  callable|null  $default
```

Comment:
Shouldn' `$default` be `mixed`?

Reply:
something like `callable|null` or `string|int` helps you know more what types does this method accept. mixed doesn't make it clear